src/types/connector.ts

```diff
@@ -238,3 +238,247 @@ export interface AppleShowTrendsFollowersDay {
     gained: number
     lost: number
 }
+
+export interface AnchorColumnHeader {
+    name: string
+    type: 'integer' | 'string' | 'number'
+    isDateTime?: boolean
+}
+
+export interface RawAnchorAggregatedPerformanceData {
```

Comment:
"Raw", because it contains columns and rows, which we simply fetch from the API without any post-processing. I couldn't come up with a better term for it.